python/services/requestservice/dmod/requestservice/service.py

```diff
@@ -170,6 +175,10 @@ async def listener(self, websocket: WebSocketServerProtocol, path):
                 event_type = MessageEventType.INVALID if req_message is None else req_message.get_message_event_type()
 
                 if isinstance(req_message, LaunchEvaluationMessage) or isinstance(req_message, OpenEvaluationMessage):
```

Comment:
`if isinstance(req_message, (LaunchEvaluationMessage, OpenEvaluationMessage))` instead of `if isinstance(req_message, LaunchEvaluationMessage) or isinstance(req_message, OpenEvaluationMessage)` might read better here.